src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -387,6 +391,18 @@ const PlrDaoStakingTransactionBlock = ({
     receiverAddress,
   ]);
 
+  const remainingSelectedFromAssetBalance = useMemo(() => {
+    const multiCallCarryOver = multiCallData?.value || 0;
+    if (!selectedFromAsset?.balance || selectedFromAsset.balance.isZero()) return 0 + multiCallCarryOver;
+    if (!amount)
+      return +ethers.utils.formatUnits(selectedFromAsset.balance, selectedFromAsset.decimals) + multiCallCarryOver;
+    const assetAmountBN = ethers.utils.parseUnits(amount, selectedFromAsset.decimals);
+    return (
+      +ethers.utils.formatUnits(selectedFromAsset.balance.sub(assetAmountBN), selectedFromAsset.decimals) +
```

Comment:
too long, add part of this or all line as const before return and then return the sum of const and `multiCallCarryOver`